Procfile.dev

```diff
@@ -1,3 +1,3 @@
-web: bin/rails server -p 3000
+#web: bin/rails server -p 3000
```

Comment:
why we commented this line?

Reply:
to separate bin/dev and rails s so debugging is easier